.github/workflows/update-cmo-jsonnet-deps.yaml

```diff
@@ -0,0 +1,58 @@
+name: Upgrade cmo jsonnet dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1'
+env:
+  USER: 'github-actions[bot]<github-actions[bot]@users.noreply.github.com>'
+
+jobs:
+  jsonnet-update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: openshift/cluster-monitoring-operator
+        ref: master
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: update jsonnet dependencies
+      run: make update
+    - name: generate yaml assets
+      run: make generate
+    - name: get pr creation app token
+      id: pr
+      uses: getsentry/action-github-app-token@v1
+      with:
+        app_id: ${{ secrets.APP_ID }}
+        private_key: ${{ secrets.APP_PRIVATE_KEY }}
+    - name: get cloner app token
+      id: cloner
+      uses: getsentry/action-github-app-token@v1
+      with:
+        app_id: ${{ secrets.CLONER_APP_ID }}
+        private_key: ${{ secrets.CLONER_APP_PRIVATE_KEY }}
+    - name: Render template
```

Comment:
In this case, I'm not sure that the pull request template is useful. We can probably skip this.

Reply:
Ack.